subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -860,15 +860,16 @@ The filename extension for the archive. By default, this is set based on the arc
 <<#sub:sharing_copy_specs,As described earlier>>, you can use the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:copySpec(org.gradle.api.Action)[Project.copySpec(org.gradle.api.Action)] method to share content between archives.
 
 [[sec:reproducible_archives]]
-=== Reproducible archives
+[[sec:reproducible_builds]]
```

Comment:
could you leave the anchor the same? There are some old links (from Gradle guides and maybe blog posts) that use it still.

Reply:
I hoped that two `sec:` are supported (the old `reproducible_archives` and the new `reproducible_build`). But this does not seem to work. Thus, I dropped the newly introduced one.